spaceship/lib/spaceship/tunes/app_export_compliance.rb

```diff
@@ -0,0 +1,46 @@
+module Spaceship
+  module Tunes
+    # Represents the encryption export compliance information for an app
+    class AppExportCompliance < TunesBase
+      # @return
+      attr_reader :id
+      attr_reader :upload_date
+      attr_reader :code_value
+      attr_reader :platform
+      attr_reader :status
+      attr_reader :export_compliance_compliance_required
+      attr_reader :uses_non_exempt_encryption_from_plist
+      attr_reader :ccat_file
+      attr_reader :builds
+      attr_reader :store_versions
+
+      attr_mapping({
+        'id' => :id,
+        'uploadDate' => :upload_date,
+        'codeValue' => :code_value,
+        'platform' => :platform,
+        'status' => :status,
+        'exportComplianceRequired' => :export_compliance_compliance_required,
```

Comment:
Just a quick double-check: Is the doubled "compliance" in `export_compliance_compliance_required` intended?